packages/teams-js/src/public/app.ts

```diff
@@ -747,8 +747,7 @@ export namespace app {
     // This will always be the current script because browsers load and execute scripts in order.
     // Whenever a script is executing for the first time it will be the last script in this array.
     const currentScriptSrc = scripts && scripts[scripts.length - 1] && scripts[scripts.length - 1].src;
-    const scriptUsageWarning =
-      'Today, teamsjs can only be used from a single script or you may see undefined behavior. This log line is used to help detect cases where teamsjs is loaded multiple times -- it is always written. The presence of the log itself does not indicate a multi-load situation, but multiples of these log lines will. If you would like to use teamjs from more than one script at the same time, please open an issue at https://github.com/OfficeDev/microsoft-teams-library-js/issues';
+    const scriptUsageWarning = `Today, teamsjs version: ${version} can only be used from a single script or you may see undefined behavior. This log line is used to help detect cases where teamsjs is loaded multiple times -- it is always written. The presence of the log itself does not indicate a multi-load situation, but multiples of these log lines will. If you would like to use teamjs from more than one script at the same time, please open an issue at https://github.com/OfficeDev/microsoft-teams-library-js/issues`;
```

Comment:
>version: ${version}  [](http://example.com/codeflow?start=47&length=20)

Rather than putting this in the script usage warning string, can we just log this version as a separate log line? It can even be written from in this function.

I just worry that if we put it in the warning string, then when we remove the warning string someday (we will eventually maybe support multiple instances) the string will be removed and the version log will go with it.

Reply:
@AE-MS, I started another thread of conversation with Saurabh and Trevor. Saurabh has more concern from PM side and I will keep discussing with them about what to do. So far, we summarized the problem as difficulty of discoverability and had some of ideas but not sure which path we will go to.